### PR TITLE
Feat: Added history managment

### PR DESCRIPTION
**Pull request by Asuraking1n**
Added Watch History
 Remove from History
 Clear history

### CODE REVIEW

**Review by Pratham82 — COMMENTED** [review]
Sign up and login card be less wider, just a suggestion. I was getting an error while logging in with dummy data, Hope so it's only the issue with preview deploy in netlify. Apapt from that looks good.

**Comment by Asuraking1n** [comment]
> Sign up and login card be less wider, just a suggestion. I was getting an error while logging in with dummy data, Hope so it's only the issue with preview deploy in netlify. Apapt from that looks good.

Yes it may be netlify issue, I will check again

File: src/pages/WatchHistory/History.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                                        />
+                                    </div>
+                                </>
+                            )}
+                        </div>
+                    </>
+                ) : (
+                    <>
+                        <div className="login-user">
+                            {" "}
```
Comment (sakshi006) [inline src/pages/WatchHistory/History.js]: You can remove these extra braces

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

**Pull request merged**